diablo/api/util.py

```diff
@@ -44,36 +40,36 @@ def _admin_required(*args, **kw):
     return _admin_required
 
 
-def put_approvals_and_scheduled(courses):
-    term_id = app.config['CURRENT_TERM_ID']
-    section_ids = [course['sectionId'] for course in courses]
-
-    approvals = Approval.get_approvals_per_section_ids(section_ids=section_ids, term_id=term_id)
-    approvals_per_section_id = objects_to_dict_organized_by_section_id(objects=approvals)
-
-    scheduled = Scheduled.get_scheduled_per_section_ids(section_ids=section_ids, term_id=term_id)
-    scheduled_per_section_id = objects_to_dict_organized_by_section_id(objects=scheduled)
-
-    for course in courses:
-        section_id = int(course['sectionId'])
-        # Approvals
-        course['approvals'] = []
-        approvals = approvals_per_section_id.get(section_id, [])
-        if len(approvals) == 1:
-            # If possible, take advantage of caching-per-UID
-            uid = approvals[0].approved_by_uid
-            calnet_users = {uid: get_calnet_user_for_uid(app, uid)}
-        else:
-            calnet_users = get_calnet_users_for_uids(app, [a.approved_by_uid for a in approvals])
-        for approval in approvals:
-            course['approvals'].append({
-                **approval.to_api_json(),
-                **{
-                    'approvedBy': calnet_users[approval.approved_by_uid],
-                },
-            })
-        # Scheduled
-        course['scheduled'] = [s.to_api_json() for s in scheduled_per_section_id.get(section_id, [])]
+# def put_approvals_and_scheduled(courses):
+#     term_id = app.config['CURRENT_TERM_ID']
+#     section_ids = [course['sectionId'] for course in courses]
+#
+#     approvals = Approval.get_approvals_per_section_ids(section_ids=section_ids, term_id=term_id)
+#     approvals_per_section_id = objects_to_dict_organized_by_section_id(objects=approvals)
+#
+#     scheduled = Scheduled.get_scheduled_per_section_ids(section_ids=section_ids, term_id=term_id)
+#     scheduled_per_section_id = objects_to_dict_organized_by_section_id(objects=scheduled)
+#
+#     for course in courses:
+#         section_id = int(course['sectionId'])
+#         # Approvals
+#         course['approvals'] = []
+#         approvals = approvals_per_section_id.get(section_id, [])
+#         if len(approvals) == 1:
+#             # If possible, take advantage of caching-per-UID
+#             uid = approvals[0].approved_by_uid
+#             calnet_users = {uid: get_calnet_user_for_uid(app, uid)}
+#         else:
+#             calnet_users = get_calnet_users_for_uids(app, [a.approved_by_uid for a in approvals])
+#         for approval in approvals:
+#             course['approvals'].append({
+#                 **approval.to_api_json(),
+#                 **{
+#                     'approvedBy': calnet_users[approval.approved_by_uid],
+#                 },
+#             })
+#         # Scheduled
+#         course['scheduled'] = [s.to_api_json() for s in scheduled_per_section_id.get(section_id, [])]
```

Comment:
@johncrossman Temporarily commented out or going away for good?

Reply:
Ah, that should go away. I'll amend.

Reply:
Amendment is in.